src/applications/check-in/day-of/pages/CheckIn/DisplayMultipleAppointments.jsx

```diff
@@ -15,24 +16,33 @@ import { createAnalyticsSlug } from '../../../utils/analytics';
 import { sortAppointmentsByStartTime } from '../../../utils/appointment';
 
 const DisplayMultipleAppointments = props => {
```

Comment:
Would be nice to eventually rename this page component too.

Reply:
@brianseek I agree, maybe just `AppointmentList`?